src/Commands/Utils.py

```diff
@@ -35,52 +35,42 @@ async def updateLeaderboardChannel(lbChannel: Channel) -> None:
         ))
 
 
-def blueTeamPick(mentions: List[Member], blueCap: BallChaser, orangeCap: BallChaser) -> Embed:
+def blueTeamPick(pickedPlayer: BallChaser, blueCap: BallChaser, orangeCap: BallChaser) -> Embed:
     """
         Helper function for the !pick command when blue team is picking.
 
         Parameters:
-            mentions: List[discord.Member] - The mentions in the sent message.
+            pickedPlayer: discord.Member - The player that has been picked.
+            blueCap: discord.Member - The blue team captain.
```

Comment:
Make sure to update these comments to be BallChasers and not discord.Members.

Reply:
Resolved.